spatialprofilingtoolbox/workflow/scripts/configure.py

```diff
@@ -1,48 +1,52 @@
 import argparse
-from shutil import which
 import os
 from os import getcwd
-import stat
 from os.path import isdir
 from os.path import exists
 from os.path import join
 from os.path import abspath
 from os.path import expanduser
-from os import getcwd
+import stat
 import importlib.resources
 
-import spatialprofilingtoolbox
 from spatialprofilingtoolbox import get_workflow
 from spatialprofilingtoolbox import get_workflow_names
```

Comment:
The error we got was here.
The only difference is `import spatialprofilingtoolbox` above is gone!
This matches the experimental behavior I thought I observed in the past, that led me to systematically include `import <package>` before getting anything from it.

Reply:
It seems the problem was in spatialprofilingtoolbox's __init__.py, now fixed.